RFCs/RFC-0: Lightweight RFC process.md

```diff
@@ -0,0 +1,69 @@
+
+### When to write an RFC?
+There is always the option of going straight to a pull request. However, for larger changes, it might be wise to de-risk the risk of rejection of the pull request by first gathering input from the community.
+
+It’s encouraged to write an RFC for any major change. A major change might be:
+* Addition of any major new feature or subsystem
+* Changes that impact existing, or introduce new public APIs.
+* Changes that need to be coordinated across multiple repositories.
+
+### How to write an RFC?
+1. Copy the RFC template (in same folder as this document) and write your proposal! It's up to the author's discretion to decide which section in the template make sense for their proposal. It's recommended to cover the problem the proposal is solving, who it affects, how you’re proposing to solve it, and answers to frequently asked questions. Explicitly listing the goals will also make it easier to retrospect later on the proposal.
+2. Add your RFC to the riff/rfcs source directory, update to the next unique number. 
+3. Post a PR for your RFC prefixing the title with `RFC-#`, where `#` is the number of your RFC. Make sure to state the deadline clearly in your PR. 
+4. Answer questions and concerns on the PR. Consider adding questions that get asked more than once to the FAQ section of the RFC.
+5. After the deadline for feedback has been reached summarize the consensus and your decision on the PR  thread. 
+    1. If the decision is to drop the proposal, the status should be updated to Dropped. 
+    2. If we are moving forward the status goes to *Active*
+    3. When there is a newer RFC that replaces this one the status goes to *Superseded* and the *Superseded By* gets updated with the number of the new RFC.
+
+### Humble Advice
+Some things can be helpful to keep in mind when writing technical documents:
+
+1. Keep the document brief but complete. People don’t have time to thoroughly read and think about extremely long documents and they’ll receive less feedback compared with a shorter document. If you find it challenging to meet this limit then maybe the proposal is too big and could be broken up.
+2. Include evidence of the problem if at all possible, even if it’s anecdotal. This can help others see the core causes of the issue rather than only being able to comment on the diagnosis or solution. For example, consider linking to evidence inline, brief inline quotes, and/or footnotes.
+3. IETF RFCs you may see contain strict rules conveyed within the semantic meaning of *SHOULD*, *MUST*, and *MAY*. You don’t need to stress about the particulars of language or semantics when writing riff-RFCs. Place importance explaining your problem and proposal clearly, succinctly, and convincingly over the specifics of implementation.
```

Comment:
```suggestion
3. IETF RFCs you may see contain strict rules conveyed within the semantic meaning of *SHOULD*, *MUST*, and *MAY*. You don’t need to stress about the particulars of language or semantics when writing riff-RFCs. Focus on explaining your problem and proposal clearly, succinctly, and convincingly rather than going into implementation detail.
```